Add files via upload

Karmotrine opened the pull request:
Changes Includes:
- Modify: `README.md` for comprehensive guide about the project + contributing

Karmotrine commented:
@Lemon1903 Follow up for approval

Lemon1903 reviewed (changes requested):
May mga areas na di pa naayos. Please refer to the last review about the areas.

Clarification lang, diba dash siya sa branch name di underscores?

Lemon1903 commented on README.md:
```diff
@@ -34,58 +38,76 @@ You can start editing the page by modifying `src/pages/index.tsx`. The page auto
-`[task-desc]` should be 2-3 words, connected by underscores / `_` , describing your task.  Example would be: `latest-news-section`
 
-4. Make sure to know what branch you are currently working on by entering: 
+IV. `[task-desc]` should be 2-3 words, connected by underscores / `_` , describing your task. Example would be: `latest_news_section`
```
diba das din to? di underscore?

Karmotrine commented on README.md:
1.  Corrections 
> inserting a change directory command in "Getting Started" section

> the [task-desc] in line 68 should use dash and not underscores. Also it becomes inline with the refactor list item, so move it down to a new line

> the pull request in line 82 should be base:dev not base:main

> the fonts folder should not be expanded in "File Component Structure" section

Applied corrections for all of these.

> in line 92, what follows https://github.com/see?
Applied corrections for this, was meant to link it to specific sections of `README.md`



2. Suggestions to add:

> I think we should introduce shadcn also here? Maybe just mention it or add it in the "Learn More" section?

You mean `Radix-UI`? Anyways, I added information about `Radix-UI`

> the "File Component Structure" section should have a short description why is it structured like that I guess? Something like the folders under 'components' folder represents the pages separation, then under those folders are sections of that page, then lastly folders under those sections are components/utils/hooks/context for that section? Just only one context is enough to understand

Added this under `File Component Structure`

Karmotrine commented:
1.  Corrections 
> inserting a change directory command in "Getting Started" section

> the [task-desc] in line 68 should use dash and not underscores. Also it becomes inline with the refactor list item, so move it down to a new line

> the pull request in line 82 should be base:dev not base:main

> the fonts folder should not be expanded in "File Component Structure" section

Applied corrections for all of these.

> in line 92, what follows https://github.com/see?
Applied corrections for this, was meant to link it to specific sections of `README.md`



2. Suggestions to add:

> I think we should introduce shadcn also here? Maybe just mention it or add it in the "Learn More" section?

You mean `Radix-UI`? Anyways, I added information about `Radix-UI`

> the "File Component Structure" section should have a short description why is it structured like that I guess? Something like the folders under 'components' folder represents the pages separation, then under those folders are sections of that page, then lastly folders under those sections are components/utils/hooks/context for that section? Just only one context is enough to understand

Added this under `File Component Structure`

Lemon1903 reviewed (approved):
Just added a one commit fixing the formatting of one part only. Anyways, LGTM!
